tests/tests.yaml

```diff
@@ -1,11 +1,7 @@
 tests: "*"
-# Dont test the beats stack until they have passing CWR tests in their own
-# bundle
+# Exclude individual charm tests, only run bundle tests
 excludes:
-  - filebeat
-  - topbeat
-  - elasticsearch
-  - kibana
+  - 'cs:'
```

Comment:
Yeah, this skips tests from any target containing `cs:` in it, and the targets happen to look like this:
```
cs:~containers/kubernetes-master
cs:~containers/kubernetes-worker
...
bundle
```